<h4>Hemispherical Bowl Draining</h4>

A hemispherical bowl has top radius $4$ feet and at time $t=0$ is full of water. At that moment a circular hole with diameter $1$ inches is opened in the bottom of the tank. How long will it take for all the water to drain from the tank?

<img src="./diagrams/hemitank.png" width="20%" height="20%">

Torricelli's law: $$\frac{dV}{dt}=A(y)\frac{dy}{dt}=-a\sqrt{2gy}$$

$$\frac{dV}{dt}=\pi\cdot(8y-y^{2})\frac{dy}{dt}=-\frac{8\pi\sqrt{y}}{576}$$

Therefore, $$72y^{\frac{3}{2}}\left(\frac{2}{5}y-\frac{16}{3}\right)=t+C$$

In [2]:
using SymPy

In [3]:
@vars y

(y,)

In [4]:
f(y)=72*y^(3/2)*((2/5)*y-(16/3))

f (generic function with 1 method)

In [5]:
f(4)

-2150.3999999999996

This means $C=-2150.40$ and $f(0)=2150.40$. It takes slightly less than $36$ minutes for the tank to drain.